howtos/documentation-guidelines.md

```diff
@@ -13,15 +13,24 @@ In the spirit of "Always Progress", if you are confident you know what change ne
 - Name Markdown files according to the title. This makes it easier to find a file. Example: **Introduction to Camunda Platform 8** --> `introduction-to-camunda-platform-8.md`. Use the file name as internal document id to reference in `sidebars.js`.
 - Sub categories have to be placed in their own sub directories. Example: Guides/Getting Started can be found in `docs/guides/getting-started`.
 
+## Versions
+
+- The [`/versioned_docs/version-*` source folders][versioned-source] contain documentation for specific releases.
+- The [`/docs/` source folder][next-source] contains documentation for the "Next" release.
+- When edits apply to the current version _and beyond_, they must be made in both the [`/docs/` folder][next-source] and the most recent [`/versioned_docs/version-*` folder][versioned-source].
+
 ## Adding a new documentation page
 
 1. Select the corresponding directory.
-2. Add the document id to `sidebars.js`.
+2. Add the document id to the corresponding sidebars file:
```

Comment:
Specifically do you mean "corresponding `sidebar.js` files"?

Reply:
not quite -- the subtle difference between lines 27 and 28 is that the versioned sidebars files are `.json`, while the next sidebars file is `.js`.